Makefile

```diff
@@ -44,11 +44,17 @@ clean-pyc: ## Remove Python file artifacts.
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -name '*~' -exec rm -f {} +
 
-sdist: ## Builds package version
-	rm dist/* ; python setup.py sdist
+clean-dist: ## Clean the dist folder
+	rm dist/*
 
-publish: sdist ## Publishes a new version to pypi.
+sdist: ## Builds package version source distribution
+	python setup.py sdist
+
+bdist_wheel: ## Builds package version wheel distribution
+	python setup.py bdist_wheel
```

Comment:
For me `python setup.py sdist bdist_wheel` does both at the same time, and I don’t think I’d ever want to do one without the other, so would prefer the single command.

Reply:
Sure thing! :+1: